pipeline-runner/tests/testthat/test-gem2s-2-load_user_files.R

```diff
@@ -443,3 +443,129 @@ test_that("parse_rhapsody_matrix uses RSEC if DBEC corrected counts are missing"
 
   expect_equal(values, expected_values)
 })
+
+
+test_that("read_10x_files removes rows with empty feature names both in count matrix and annotation if present and < 0.1%", {
+  counts <- mock_counts()[rep(seq_len(nrow(mock_counts())), each = 10), ]
```

Comment:
why is this necessary? maybe add a comment

Reply:
this is to increase the size of counts matrix, to be able to mock a matrix with < 0.1% of empty features. I'll add a comment